launcher.py

```diff
@@ -410,7 +410,7 @@ def update_watcher_postgres_database(self) -> bool:
         '''
         os.chdir(os.path.expanduser('~') + '/elixir-omg')
         result = subprocess.run(
-            'mix ecto.migrate --no-start',
+            'mix ecto.migrate',
```

Comment:
Hey, if you wrap them in double quotes `"mix ecto.migrate --no-start"` or 
if you put the args in a list `['mix', 'ecto.migrate', '--no-start']` it'll start to work. While you're at it, same problem is in `initialise_watcher_postgres_database` -> `'mix ecto.reset --no-start'`